scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -586,6 +588,36 @@ class ScioContext private[scio] (val options: PipelineOptions,
     }
 
   /**
+    * Get an SCollection for a CloudSql query
+    * @group input
+    */
+  def cloudSqlSelect[T: ClassTag](readOptions: JdbcReadOptions[T]): SCollection[T] = {
```

Comment:
CloudSQL nothing but mysql, therefore this is just a wrapper method which call `jdbcSelect` inside it. Thought good to have this explicit method to use with CloudSql which looks like different from jdbc at the first sight.